functions/api/image/tokens/[[index]].tsx

```diff
@@ -147,7 +147,7 @@ export const onRequest: PagesFunction = async ({ params, request }) => {
                     marginLeft: '-13px',
                   }}
                 >
-                  {data.symbol}
+                  {data.symbol.slice(0, 6)}
```

Comment:
to solve the problem of the symbol not crossing two lines can't you just set the overflow here? to like overflow none, and text-wrap to ellipsis or will that not work for some reason?

Reply:
i did not think of that LOL

Reply:
LOL